### DISCUSSION TITLE
Unique IDs for repeated session for Eventfahrplan

### DISCUSSION THREAD

**Discussion by navlisilvan** [pretalx/pretalx#1572] [Q&A]
Hi,
Last year we tried to use Pretalx with the Eventfahrplan and encountered the problem that the app has problems with repeating talks and only shows the first of them.
https://github.com/EventFahrplan/EventFahrplan/issues/496
One suggested solution was to have Pretalx "spit out" Unique IDs for each Talk. 

Is this a possible solution from your side or is it not even up for discussion?

Thanks a lot!

**Comment by rixx** [comment]
Hard to tell what was going on, as you've taken down / redirect away from the 2022 event. pretalx does provide an event GUID in the schedule.xml that is meant to be unique even across repeat events, and has been tested to work. Without the actual broken schedule.xml and further information (which talk slots are concerned, which pretalx version was running), I don't think I can help.

**Reply by navlisilvan** [reply]
Thx, i try to "repair" the App and I am writing again with all the necessary information.
